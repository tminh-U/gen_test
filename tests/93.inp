10
7 9 17 2 7 3 14 6 10 8 
15
4 5
2 8
6 6
4 7
3 9
8 8
7 10
2 6
1 6
2 7
1 9
2 6
2 10
8 10
2 6
