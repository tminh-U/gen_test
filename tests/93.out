4
20
2
12
27
2
7
9
14
16
42
9
41
6
9
